src/sync/incomingSyncUtils.js

```diff
@@ -701,6 +701,8 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
         period,
         orderType: record.orderType,
         customData: parseJsonString(record.custom_data),
+        isRemoteOrder: parseBoolean(record.isRemoteOrder),
```

Comment:
I guess all past requisitions will default to false? Probably get away with not resyncing them; I think it would only be an issue for Myanmar if they put an existing desktop site onto mobile before the release of this feature... which would likely problematic before upgrading anyway, lol.

Reply:
Yeah - those were my thoughts also. Not 100% convinced it's the best solution, if you think otherwise.